smaily-for-wordpress.php

```diff
@@ -233,20 +246,34 @@ function smaily_nojs_subscribe_callback() {
 
 	$array = array_merge( $array, $form_values );
 	require_once( BP . DS . 'code' . DS . 'Request.php' );
-	$request = new Wp_Sendsmaily_Request( $server, $array );
+	$request = new Wp_Smaily_Request( $server, $array );
 	$result  = $request->exec();
 
 	if ( empty( $result ) ) {
-		echo esc_html__( 'Something went wrong', 'wp_sendsmaily' );
+		$redirect_url = add_query_arg( 'smaily_form_error',
+			rawurlencode( esc_html__( 'Something went wrong', 'wp_smaily' ) ), $redirect_url );
```

Comment:
Replace `esc_html__( ... )` with `__( ... )`. Otherwise string is encoded twice: once here and when template renders the error message.